src/components/menu/Menu.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { CSSTransition } from 'react-transition-group';
+
+import './Menu.css';
+import iconOpen from '../../assets/menu_icon_open.svg';
+import iconClose from '../../assets/menu_icon_close.svg';
+
+const Menu = () => {
```

Comment:
You could memo this component. 
https://reactjs.org/docs/react-api.html#reactmemo